### ISSUE TITLE
Fix problem with loading checkpoint of postprocessor

### ISSUE THREAD

**Issue by DaloroAT** [OML-Team/open-metric-learning#300]
The problem appears because of the hack when we add link to postprocessor to LightningModule's field so it can handle model's device.

Temporary workaround:

```python

from pathlib import Path

import torch

path_to_ckpt = Path("...")

y = torch.load(path_to_ckpt, map_location="cpu")["state_dict"]

kvs = list(y.items())
for k, v in kvs:
    if k.startswith("model_pairwise"):
        del y[k]

path_to_save = path_to_ckpt.parent / "best_fixed.ckpt"
torch.save({"state_dict": y}, path_to_save)


```
